### PR TITLE
Added parent post and order ID to a bunch of Tribe Commerce email filters

### PR DESCRIPTION
**Pull request by niconerd**
https://central.tri.be/issues/104209

Also added docblocks for all filters

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Whitespace found at end of line | [Main.php:993](https://github.com/moderntribe/event-tickets/blob/61bc886cbf84f4d7fb2881c9548bf1893d67ea3a/src/Tribe/Commerce/PayPal/Main.php#L993)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Whitespace found at end of line | [Main.php:993](https://github.com/moderntribe/event-tickets/blob/4c9494e7d82e5b0ce15b4c8e701620abb844638f/src/Tribe/Commerce/PayPal/Main.php#L993)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Commerce/PayPal/Main.php [hunk 1]
```diff
@@ -987,7 +987,10 @@ public function generate_tickets( $payment_status = 'completed', $redirect = tru
 	/**
 	 * Sends ticket email
 	 *
+	 * @since TBD added $post_id parameter
+	 *
 	 * @param int $order_id Order post ID
+	 * @param int $post_id  Parent post ID
 	 */
 	public function send_tickets_email( $order_id ) {
```
Comment (bordoni) [inline src/Tribe/Commerce/PayPal/Main.php]: We need receiving any second param here. Make sure we make it optional, since older code will break otherwise.

**Comment by niconerd** [comment]
@bordoni thanks for the review buddy. Updated accordingly.

**Pull request merged**